### ISSUE TITLE
:winopen does not work

### ISSUE THREAD

**Issue by Yuhta** [vimperator/vimperator-labs#781]
##### Issue type:

- Bug report

##### Version:

```
Vimperator:	3.16.0 (created: 2017/02/01 00:55:15)
Firefox:	Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:55.0) Gecko/20100101 Firefox/55.0
```

##### Description:

Nothing happens after :winopen

##### Expected behavior:

New window created after :winopen

**Comment by rdgoite** [comment]
I am experiencing the same problem.

#### Version:
```
Vimperator:	3.16.0 (created: 2017/02/01 00:55:15)
Firefox:	Mozilla/5.0 (Macintosh; Intel Mac OS X 10.11; rv:55.0) Gecko/20100101 Firefox/55.0
```

**Comment by SammysHP** [comment]
Already fixed in ae0df8040bc2821d6d0ce39a9c2f3b8feda4a3e4.
